sdks/python/container/Dockerfile

```diff
@@ -51,16 +51,20 @@ RUN ln -s /usr/bin/ccache /usr/local/bin/gcc
 RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=false
 
 COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
+ADD target/license_scripts /tmp/license_scripts/
```

Comment:
COPY is preferred over ADD. Can we remove usages of ADD to reduce cognitive load on people who might think why are we using sometimes COPY and sometimes ADD?
See: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy